src/raster/r.forcircular/r.forcircular.html

```diff
@@ -44,6 +44,112 @@ <h2>NOTES</h2>
+In forest area where extraction is possible, the stumpage value is quantified as a combination of differenct factors such as hourly unitary cost of each production process, worker costs, hourly productivity and administrative costs. <br>
+The user can setthe upper and – if needed – lower limits for slope and distance from roads to obtain the technical surface: <br>
+<img src="./tech_function.png" alt=""><br/><br/>
+In forest area where extraction is possible, the stumpage value is quantified as follow: <br/>
+<img src="./stumpage_function.png" alt=""><br/><br/>
+Subsequently, the 4R indicators are calculated on the forest area with positive revenue. <br> <br>
+The importance of each circular bioeconomy index derives from the work of Paletto et al. (see Paletto A, Becagli C, Geri F, et al (2022) Use of Participatory Processes in Wood Residue Management from a Circular Bioeconomy Perspective: An Approach Adopted in Italy. Energies 15:1011) where – starting from a total sample of 56 decision makers operating in forest-based sector in Italy – 30 decision makers filled out a questionnaire. The decision makers involved in the study have been identified based on their knowledge and experience in the fields of bioeconomy, circular economic, and forest policy. According to the outputs provided by Paletto et al., the results currently applied in <em>r.forcircular</em> (but modifiable by users) are: i1=0.15, i2=0.12, i3=0.12, i4=0.13, i5=0.14, i6=0.17, i7=0.16. <br> <br>
+<em>r.forcircular</em> highlights the output in both numerical and geographical format. By means of zonal statistics operations following results can be reported for the forest surface where stumpage value is greater then 0: annual availability of assortments (m<sup>3</sup>/y) recalibrated in category of harvested material (i.e., roundwood, timber pole, firewood, woodchips converted in bioenergy and expressed in MWh/y), annual stumpage value (&euro;/y), average annual stumpage value (&euro;/ha·y-1), average stumpage value at harvesting (&euro;/ha) and avoided CO<sub>2</sub> emissions (tCO<sub>2</sub>/y). <br> <br>
+Quantification of circular bioeconomy is expressed by a SMCA procedure based on compromise programming (CP) technique. CP depicts the distance from the so-called “ideal” point, a hypothetical alternative defined as the most suitable level for each indicator (i) in the considered scenario. The distance from ideal point (DIP) is measured with the decision rule: <br>
+<img src="./dip.png" alt="">
+
+
```

Comment:
Would you mind  please formatting all the text that is not within tables in ~80 characters lines? It's common practice and makes it so much easier to read :pray:

Reply:
Done